hydra-node/src/Hydra/API/Server.hs

```diff
@@ -50,15 +64,17 @@ runAPIServer ::
   IP ->
   PortNumber ->
   Tracer IO APIServerLog ->
+  TVar [Either (ClientInput tx) (ServerOutput tx)] ->
```

Comment:
Lovely use of `Either` though :+1:

Reply:
It somehow always disturbs me to use `Either` when the first type isn't an error-like type :sweat_smile: ...

Reply:
I can relate.. it makes you feel a bit sad for the first type .. as it is being "not the `Right` one"